modules/openy_gc_shared_content/src/Form/SettingsForm.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Drupal\openy_gc_shared_content\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Class SettingsForm.
+ */
+class SettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'openy_gc_shared_content_settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('openy_gc_shared_content.settings');
+
+    $form['shared_content_server'] = [
+      '#type' => 'url',
```

Comment:
it should be multiple fields like we have at 12busrts schedule or one textarea with many lines